packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -43,19 +63,28 @@ export default class BookReaderWrapper extends Component {
       },
     };
     const fullOptions = {
-      ...defaultOptions,
-      ...options,
       defaultStartLeaf: 0,
       titleLeaf: 0,
+      ...defaultOptions,
+      ...options,
     };
-    const br = new BookReader(fullOptions);
-    window.br = br;
-    br.init();
+    // There are two ways to initialize BookReader, eithr through JSIA which includes IA specific lending info
+    // or the simpler initialization without IA.
+    if (this.props.jsia) {
+      BookReaderJSIAinit(this.props.jsia, fullOptions); // Creates window.br
```

Comment:
is this a reference to the main site's `BookReaderJSIA.js`?

Reply:
Yes - this is the principle point of the PR, allow (optionally) passing in as props, the jsia returned by the main site's API and then to use the appropriate initializer in BookReaderJSIA.js  ..

Reply:
Happy to, but sorry can you point me at instructions to run Storybook - I've had it shown to me, but don't know how to start it up.

Reply:
this should do it: `cd packages/ia-components && yarn run storybook`

if storybook isn't available, just yarn.

Reply:
Thanks - I was looking for an executable and not finding it :-)
Yes - that worked, it comes up on Page 2, is that correct ?

Reply:
should look like this, except with Willie Nelson:

![image](https://user-images.githubusercontent.com/7840857/65919848-5ad2b700-e392-11e9-8094-3085591202cf.png)

Reply:
Yes - it does, except it comes up on page 2, and I'm wondering if it is supposed to come up on page 1

Reply:
Yes, it should start on page 1.
Is it because these are not set?
```
defaultStartLeaf: 0
titleLeaf: 0
```

Reply:
Its because the caller of BookReaderWrapper is explicitly passing in titleLeaf:1 defaultStartLeaf: 1  ... Thats coming from TheatreAudioPlayer.props.linerNotes.brOptions  
which I think is coming from album-private-with-spotify-youtube.js (i.e. your test data) 

I.e. I think that the Liner Notes go to page 2 (assuming that this is a 0-basd count) with the new BookReaderWrapper, and only go correctly to page 1, when the Component overrides your bad (I think) test data. 

I could be missing something of course .

Reply:
thanks for resolving, @mitra42